src/appleseed/foundation/math/microfacet.h

```diff
@@ -660,22 +491,85 @@ class BeckmannMDF
         return slope;
     }
 
-    typedef boost::math::policies::policy<
-        boost::math::policies::promote_float<false>,
-        boost::math::policies::digits10<6>
-    > ErfPolicyType;
-
-    static T erf(const T x)
+    virtual T pdf(
+        const Vector<T, 3>&  v,
+        const Vector<T, 3>&  h,
+        const T              alpha_x,
+        const T              alpha_y) const APPLESEED_OVERRIDE
     {
-        return boost::math::erf(x, ErfPolicyType());
+        return MDF<T>::pdf_visible_normals(*this, v, h, alpha_x, alpha_y);
     }
 
-    static T erf_inv(T x)
-    {
-        if (std::abs(x) == T(1.0))
-            x -= boost::math::copysign(T(1.0e-5), x);
+  private:
+
+    //
+    //  Reference:
+    //
+    //    Handbook of Mathematical Functions.
+    //    Abramowitz and Stegun.
+    //    http://people.math.sfu.ca/~cbm/aands/toc.htm
+    //
+    //  Copied from from pbrt-v3.
+
+    inline T erf(const T x) const
+    {
+        // constants
+        const T a1 = T(0.254829592);
+        const T a2 = T(-0.284496736);
+        const T a3 = T(1.421413741);
+        const T a4 = T(-1.453152027);
+        const T a5 = T(1.061405429);
+        const T p = T(0.3275911);
+
+        // A&S formula 7.1.26
+        const T abs_x = std::abs(x);
+        const T t = T(1.0) / (T(1.0) + p * abs_x);
+        const T y = T(1.0) - (((((a5 * t + a4) * t) + a3) * t + a2) * t + a1) * t * std::exp(-square(abs_x * abs_x));
+        const T sign_x = x < T(0.0) ? T(-1.0) : T(1.0);
+        return sign_x * y;
```

Comment:
return x < T(0.0) ? -y : +y;

is probably going to be at least as fast, if not faster, in all cases.